lib/basedriver/driver.js

```diff
@@ -408,6 +423,31 @@ class BaseDriver extends Protocol {
     return false;
   }
 
+  proxyRouteIsAvoided (sessionId, method, url) {
+    for (let avoidSchema of this.getProxyAvoidList(sessionId)) {
+      if (!_.isArray(avoidSchema) || avoidSchema.length !== 2) {
```

Comment:
this will throw inside avoidSchema.length if `avoidSchema` is null

Reply:
existing code (moved from protocol) but good point, i'll fix

Reply:
actually, this is OK. if avoidSchema is not an array, the condition will short-circuit since the LHS is true, and `avoidSchema.length` will never be evaluated